extensions/data-transfer/portability-data-transfer-apple/src/main/proto/ImportPhotos.proto

```diff
@@ -58,6 +58,7 @@ message NewMediaRequest {
   optional string encodingFormat = 6;
   optional uint64 creationDateInMillis = 7;
   optional string singleFileUploadResponse = 8;
+  optional uint64 isFavorite = 9;
```

Comment:
could you switch to [protobuf's `bool` type](https://protobuf.dev/programming-guides/proto2/#scalar)?

otherwise, maybe just document this, eg: "boolean delivered as int; will always be 1 or 0."

Reply:
+1

Reply:
> otherwise, maybe just document this, eg: "boolean delivered as int; will always be 1 or 0."

Done